### DISCUSSION TITLE
Gwenno's backpack of Junk

### DISCUSSION THREAD

**Discussion by Tygaran** [exult/exult#430] [General]
It was suggested that I cross post this oddity over here.

Here's an odd question.  At the end of the game, when I'm able to recruit Gwenno into the party, her backpack is always filled with junk.  However, neighter her backpack nor bag has any contens within the npc record.  Does that occur during the original game?  What's she doing; stealing things while affected by the Wantonness Bane? =]

**Comment by KnightCaptainU7** [comment]
I think some of her activities post-Banes involves her doing things like DESK_WORK which will generate new items. What items is she carrying?

**Reply by Tygaran** [reply]
Most recent runhad: 3 parchment, two seal, 3 quillholder, and an inkwell.

**Comment by wench** [comment]
Those items are consistent with the Deskwork schedule and I would suggest the schedule should probably clean up after itself and not leak items, not sure if there is a straight forward way to do that though.

**Comment by KnightCaptainU7** [comment]
It may be a bug in Exult, but I like the idea of Bane-madness causing Gwenno to horde random items.
